user-guide/Basic_Functionality/Protocols_and_templates/Alarm_templates/Configuring_alarm_templates/Configuring_anomaly_detection_alarms.md

```diff
@@ -4,7 +4,74 @@ uid: Configuring_anomaly_detection_alarms
+
+   - To choose one of the preset options, click the *Select preset* selection box in the top-left corner:
+
+     - **All disabled**: No anomaly alarms are generated.
+
+     - **All smart**: Anomaly alarms are generated based on what DataMiner Analytics determines to be an anomaly.
+
+   - To customize the anomaly alarm settings:
+
+     1. Configure alarms for specific types of anomaly detection:
+
+        - *Level shift > Level increases*: Enables or disables alarms for level increase anomalies.
+
+        - *Level shift > Level decreases*: Enables or disables alarms for level decrease anomalies.
+
+        - *Outlier > Upward spikes*: Enables or disables alarms for upward spikes.
+
+        - *Outlier > Downward spikes*: Enables or disables alarms for downward spikes.
+
+        - *Variance change > Variance increases*: Enables or disables alarms for variance increases.
+
+        - *Variance change > Variance decreases*: Enables or disables alarms for variance decreases.
+
+        - *Trend change > Slope increases*: Enables or disables alarms for slope increases.
+
+        - *Trend change > Slope decreases*: Enables or disables alarms for slope decreases.
+
+        - *Flatline > Detect flatlines*: Enables or disables alarms for flatline anomalies.
+
+        > [!NOTE]
+        > If you, for example, select the checkbox next to *Slope increases* but clear the checkbox next to *Slope decreases*, an alarm will be generated when an anomalous slope increase is detected but a suggestion event will be generated if an anomalous slope decrease is detected. It is not possible to generate both an alarm and a suggestion event for the same type of anomaly detection.
+
+     1. For anomaly detection of types *Level shift* and *Outlier*, specify when an alarm should occur.
+
+        - **Smart**: Alarm thresholds are set based on what DataMiner Analytics determines to be an anomaly. This is the default option.
+
+        - **Relative**: Alarm thresholds are set as a percentage, which represents the delta with the baseline value.
+
+        - **Absolute**: Alarm thresholds are set as an absolute value, which represents the delta with the baseline value.
+
+1. Click *Close* in the lower right corner to exit the *Anomaly Alarm Settings* pop-up window.
+
+   In the alarm template editor, depending on your changes, the *Anomalies* button will now indicate the level of anomaly monitoring that has been configured:
+
+   - **Disabled**: No anomaly alarms are generated for this parameter.
+
+   - **Customized**: You have customized the anomaly alarm settings for this parameter.
+
+   - **Smart**: Anomaly alarms are generated for this parameter based on what DataMiner Analytics determines to be an anomaly.
+
+1. To save your changes, click *OK* or *Apply* in the lower right corner of the alarm template editor.
+
+## [Prior to DataMiner 10.2.0 [CU21]/10.3.0 [CU9]/10.3.12](#tab/tabid-2)
```

Comment:
I'm not sure how this is usually documented, but there are now three possibilities depending on your Cube and DataMiner version:

- Old Cube (i.e. a version older than 10.2.0 [CU21]/10.3.0 [CU9]/10.3.12) and old DataMiner (i.e. earlier than 10.3.12/10.4.0): configuration is still done as described in this last tab
- New Cube and old DataMiner: configuration also works via a pop-up here, but it only has the options that were previously available via the procedure below
- New Cube and new DataMiner: configuration works via de procedure above

Since one can also work with a different version of Cube than is included in the upgrade package (maybe it is even the default?), whether you have the second option or the third does not depend on the version of DataMiner you have installed, but only on the version of Cube.